### DISCUSSION TITLE
How is week 8 coming along?

### DISCUSSION THREAD

**Discussion by fireredfan7** [FunkinCrew/Funkin#1876] [Q&A] [answered]
I'm just curious about progress and not asking for a release date.

**Comment by AbnormalPoof** [comment, marked as answer]
it will never release

**Reply by fireredfan7** [reply]
What do you mean?

**Reply by isakube** [reply]
it will never release :trollface:

**Reply by Swapper1** [reply]
Ninja has already said that week 8 will be come out whether when full game is release or not. It doesn't has an release date yet and the progress haven't started yet.

**Reply by DeceptedPar** [reply]
ninjamuffin has said that week 7 will release with week 8
so it'll prob release sometime next year around February or something (I don't know for sure, no one does)
